main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/Floor-Gang/prevent-link-redirection/internal/discord"
+)
+
+const (
+	configPath = "./config.yml"
+)
+
+func main() {
+	err := discord.Start(configPath)
+
+	if err != nil {
+		panic(err)
+	}
+
+	keepalive()
```

Comment:
Try out the utility package. It has keepalive in it, which may be more effective than this.